src/app/modules/attachment/attachment.service.ts

```diff
@@ -0,0 +1,51 @@
+import crypto from 'crypto'
```

Comment:
I'm wondering whether this should be part of `encrypt-submission.service.ts`, since it's very specific to the encrypt submissions pipeline. what do you think?

Reply:
Makes sense, moving it over.